src/en/proposals/fairlysadpanda-revolutionaries.md

```diff
@@ -0,0 +1,333 @@
+3. If successful, the target is silently converted to a Revolutionary.
+4. The ability enters a cooldown.
+
+### Removal of MindShield implants being orderable from Cargo
+
+MindShields are metagameable and make it extremely unfun to play as a Head Rev. They have no interesting interactions; they’re just a hand-wave for why some people are completely immune to a game mechanic, and whilst that’s utterly required for round flow (allowing Sec and the Heads to trust each other) it is a magic solve-everything button which leads to a total wall in design.
+
+A small number of MindShields could be kept on-station to allow staff promotions within reason. _A kind admin might decide to send a crate if the station runs out and it'd be good for the round, but this is out of scope of this design document._
+
+### Removal of Flashes being able to convert people
+
+Flashes are metagameable, confusing for newbies, hard to balance, and enforce too much reliance on manufacture by Science by the Revs for effective round flow. They also have no narrative justification beyond it being an Ur-mode from the dawn of SS13. There are people playing SS14 that are younger than the only good _Men In Black_ movie.
+
+### Addition of Influence as a tracked player stat, and associated gameplay features
+
+Influence is a value that each player carries. MindShielded players are always value 0. Influence is raised by a few routes (exposure to Head Revs, propaganda and one or two restricted drugs like Mindbreaker Toxin).
+
+Players gain Influence by:
+
+- Standing near a Head Rev they can see.
+
+- Being near Propaganda they can see.
+
+- Being injected with Mindbreaker Toxin. This is a very big bump and is intended for kidnap scenarios.
+
+Players lose influence by:
+
+- Not being near any source of Influence. This is quite slow, and is reduced to nil if the player has been converted.
+
+- Influence degrades reasonably quickly if placed in the Sec cells and exposed to calming jazz music.
+
+### Influence ticks up when exposed to sources and slowly ticks down when not. There is a max Influence level.
+
+Addition of NT and Revolutionary posters and propaganda, that both sides, and traitors, can put up and take down across the station.
+
+These posters can be printed across the station by both sides and gives Security Cadets, Lawyers and bored Passengers something to do, even on non-Rev rounds, and a reason for Sec to patrol (imagine!). They’re an DOT effect with a max cap per item, and add or subtract Influence from those who see them.
+
+Putting up pro-Revolution posters is added as a Traitor goal to make them have a reason to appear outside Revs.
+
+Clowns get the ability to put up annoying "Visit Clown Planet" tourism posters.
+
+Posters can be be destroyed by spraying them with paint or tearing them down.
+
+### Addition of Loyalty Loudspeakers in Sec
```

Comment:
More themeatically might be to say these are "Mindshield Emitters, An experimental device utilising high power transmitters to project a small field of thought manipulation blocking rays similar yet weaker compared to existing Nanotrasen implants." or similar spess tech.

Reply:
Yeah, the fluff can be revised depending on feedback. I like the "funny" aspect - calming Revs down via a hokey solution like smooth jazz - but it depends on the implementer to come up with a specific idea and get it merged.